gridt/src/app/movements/add-movement/add-movement.page.spec.ts

```diff
@@ -20,11 +20,11 @@ class AuthServiceStub {
   readyAuthentication$ = of(default_headers)
 }
 
-describe('AddMovementPage', () => {
+describe("AddMovementPage", () => {
   let component: AddMovementPage;
   let fixture: ComponentFixture<AddMovementPage>;
   let authSpy: AuthService;
-  let alertSpy: AlertController = jasmine.createSpyObj("alertSpy", ["create", "dismiss"]);
+  const alertSpy: AlertController = jasmine.createSpyObj("alertSpy", ["create", "dismiss"]);
```

Comment:
Why is it important that it's a const?

Reply:
My linter keeps telling me that it should be constants because they never get changed. So i just accepted it's requests this time. However i think we might need some general consensus on these thngs so that the code stays uniform. Either all Lets or const if they don;t get reassigned.